src/main/java/io/github/thebusybiscuit/slimefun4/api/blocks/SlimefunBlockData.java

```diff
@@ -25,6 +25,11 @@
  */
 public class SlimefunBlockData extends AbstractDataObject {
 
+    @Nonnull
+    public static SlimefunBlockData empty() {
+        return EmptyBlockData.INSTANCE;
+    }
```

Comment:
Again, not entirely certain whether we will be needing this or if the class will be removed.
So let's rather remove it for now.